components/Movie/Moviecard.tsx

```diff
@@ -0,0 +1,96 @@
+// Functions
+import { FavoriteButton } from "components/FavoriteButton";
+
+// Styles
+import styles from "components/Movie/Movie.module.css";
+
+interface IMoviecardProps {
+  moviedata: string;
+}
+
+export default function Moviecard({ moviedata }: IMoviecardProps) {
```

Comment:
Data should not come in here as a JSON string. Make sure you make a nice interface for movie.
Also Movie just renders `Movicard. What do you need `Movie` for?